Modules/faulthandler.c

```diff
@@ -1373,6 +1378,17 @@ _PyFaulthandler_Init(int enable)
        signal handler uses more than SIGSTKSZ bytes of stack memory on some
        platforms. */
     stack.ss_size = SIGSTKSZ * 2;
+#ifdef AT_MINSIGSTKSZ
+    {
```

Comment:
These braces are not needed. Python 3.11 uses C11, see: https://peps.python.org/pep-0007/#c-dialect

Reply:
I had a question here. As is the change will only be effective on the new hardware if CPython was compiled against kernel 5.14 of later. Would be acceptable to do

```c
#ifndef AT_MINSIGSTKSZ
#define AT_MINSIGSTKSZ 51
#endif
```

relying on the fact that `getauxval` returns 0 for unknown tokens?

This would permit backport of this fix to earlier version of CPython, right?

Reply:
> These braces are not needed. Python 3.11 uses C11, see: https://peps.python.org/pep-0007/#c-dialect

Removed, and indentation adjusted. Would appreciate feedback about tweaking the proposed fix so that the call to `getauxval` is not in the preprocessor branch by defining `AT_MINSIGSTKSZ` if undefined.

Reply:
Please don't hardcode constants in Python: don't add ``#define AT_MINSIGSTKSZ 51``.

Reply:
> Please don't hardcode constants in Python: don't add `#define AT_MINSIGSTKSZ 51`.

Ok, understood, but had to ask.